src/core/server/server.api.md

```diff
@@ -763,7 +765,7 @@ export type IScopedClusterClient = Pick<ScopedClusterClient, 'callAsCurrentUser'
 export interface IUiSettingsClient {
     get: <T extends SavedObjectAttribute = any>(key: string) => Promise<T>;
     getAll: <T extends SavedObjectAttribute = any>() => Promise<Record<string, T>>;
-    getDefaults: () => Record<string, UiSettingsParams>;
+    getRegistered: () => Readonly<Record<string, UiSettingsParams>>;

```

Comment:
We can still keep `getDefaults` for BWC with 3rd party plugins. I decided we don't want to have an additional API to maintain. Any objections?